app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -158,6 +225,7 @@ async def date_conf_intensity_symbology(
 async def _merge_intensity_and_date_conf(
```

Comment:
Technically, we could use PixETL for this one now as well and get rid of the C++ script. 
We should make this a seperate PR and time the differnce between the C++ and the PixETL implementation and go from there.